src/init.cpp

```diff
@@ -74,11 +71,11 @@
 #include "recipe_dictionary.h"
 #include "recipe_groups.h"
 #include "regional_settings.h"
+#include "relic.h"
 #include "requirements.h"
 #include "rotatable_symbols.h"
 #include "scenario.h"
 #include "scent_map.h"
```

Comment:
It is required for code guarded by TILES.

```suggestion
#include "scent_map.h"
#include "sdltiles.h"
```